Paws4Me/Paws4Me/Controllers/Logic/SignInViewController.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  SignInViewController.swift
+//  Paws4Me
+//
+//  Created by Kirsty-Lee Walker on 2022/02/26.
+//
+import UIKit
+
+protocol Validation {
+    func isValidCredentials(username: String, password: String) -> Bool
+}
+
+class SignInViewController: UIViewController, Validation {
+    @IBOutlet weak private var txtUsername: UITextField!
+    @IBOutlet weak private var txtPassword: UITextField!
+    @IBOutlet weak private var btnSignIn: UIButton!
+    @IBOutlet weak private var btnRegister: UIButton!
+    var isLoggedIn = false
+    let bottomLine = CALayer()
+    let bottomLine2 = CALayer()
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        btnSignIn.addCornerRadius(btnSignIn)
+        btnSignIn.addCornerRadius(btnRegister)
+        btnSignIn.changeBorderLook(btnRegister)
+        addUnderline(txtUsername, txtPassword, bottomLine: bottomLine, bottomLine2: bottomLine2)
+    }
+    @IBAction private func btnSignInClick(_ sender: UIButton!) {
```

Comment:
Still need that enhancement for this button name.

Reply:
The previous comment highlighted the Any so I changed it to UIButton. See below:
 
 @IBAction private func btnSignInClick(_ sender: Any) {
    @IBAction private func btnSignInClick(_ sender: UIButton!) 
    
    Should I actually rename btnSignInClick to rather SignInClick?

Reply:
So as I said in another comment since we are doing mobile development what we normally refer to is `tapped` over `clicked` as well as we don't really like to use abbreviations either. My suggestion would be to rename this:

`@IBAction private func signInButtonTapped`

Reply:
Noted. I will remember this going forward. Thank you 👍🏻

Reply:
Thank you for all your feedback Brandon. I really appreciate it